src/components/LibraryContainer.tsx

```diff
@@ -69,6 +71,29 @@ export class LibraryContainer extends React.Component<LibraryContainerProps, Lib
         };
     }
 
+    componentWillMount() {
+        window.addEventListener("keydown", this.handleKeyDown.bind(this));
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener("keydown", this.handleKeyDown.bind(this));
+    }
+
+    handleKeyDown(event: any) {
```

Comment:
I've removed the keyboard event in `LibraryContainer`. The first `SearchResultItem` will update the `selectionIndex` in `LibraryContainer` on arrow up/down keyboar event now.